### PR TITLE
code block syntax fix

### PR DESCRIPTION
**Pull request by spacedsweden**

### CODE REVIEW

File: docs/sms/helperlibraries/sms-java-library.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ hidden: false
 In **Gradle**, please put the lines below in your **build.gradle**
 
 ```javascript
-implementation 'com.sinch:sdk-sms:1.0.4'
+implementation 'com.sinch:sdk-sms:1.*'
```
Comment (spacedsweden) [inline docs/sms/helperlibraries/sms-java-library.md:12]: to make sure the latest is installed over a specific version

File: docs/sms/sms.md [hunk 2]
```diff
@@ -16,10 +17,8 @@ account](https://dashboard.sinch.com/signup). If you are in the United States, y
 
 If you haven't already, create a new node app with npm, accept the defaults, and add the request package.
 
-```javascript
```
Comment (spacedsweden) [inline docs/sms/sms.md:19]: orphaned code block, messes up the whole file, we always need to make sure there is a closing ``` when opening a code block

File: docs/sms/sms.md [hunk 3]
```diff
@@ -88,15 +86,16 @@ const server = http.createServer((req, res) => {
 });
 server.listen(3000);
 ```
+
 Start the server
 
-'''
```
Comment (spacedsweden) [inline docs/sms/sms.md:93]: straight ticks dont make it to codeblocks

**Review by JEllis72 — APPROVED** [review]
I can see what I missed.

**Pull request merged**